test/architecture/src/esorigin/Dockerfile

```diff
@@ -0,0 +1,19 @@
+##
+# esorigin
+# Build seed data into our image for tests; appropriated from:
+# https://stackoverflow.com/questions/35526532/how-to-add-an-elasticsearch-index-during-docker-build
+##
+FROM elasticsearch:7.16.1
+
+RUN mkdir /data && \
+    chown -R elasticsearch:elasticsearch /data && \
+    echo 'path.data: /data' >> config/elasticsearch.yml && \
+    echo 'discovery.type: "single-node"' >> config/elasticsearch.yml && \
+    echo "xpack.security.enabled: false" >> config/elasticsearch.yml
```

Comment:
I'd add some of these here too so that ES doesn't refuse to start with high disk usage

```yaml
cluster.routing.allocation.disk.watermark.flood_stage: "99%"
cluster.routing.allocation.disk.watermark.high: "99%"
```